app/views/miq_request/_prov_field.html.haml

```diff
@@ -407,7 +407,7 @@
           - unless field_hash[:notes_display] == :hide || field_hash[:notes].blank?
             -# Display notes if available
             = field_hash[:notes]
-      - elsif [:linked_clone, :placement_auto, :stateless, :sysprep_auto_logon, :sysprep_change_sid, :seal_template,
+      - elsif [:linked_clone, :migratable, :placement_auto, :stateless, :sysprep_auto_logon, :sysprep_change_sid, :seal_template,
```

Comment:
As this line existed before your changes, it's okay to just add something, but also okay to replace the array of symbols with the `%i[]` shorthand...your call.

Reply:
Thanks for the review. I've accepted your suggestions. Concerning the last, I'd rather touch as less non-IBM related code as possible.